docs/KuberntesTargetFilter.md

```diff
@@ -0,0 +1,24 @@
+# Kubernetes target filtering
+
+The config parameter `target_discovery.filter` should be used to filter in the targets that Prometheus will scrape. Current conditions are filtering by `label` and `annotation`. All conditions are applied using an `AND` operation.
+
+Following example will scrape only `Pods` having the `newrelic.io/scrape: true` annotation and a label `k8s.io/app` with value `postgres` or `mysql`.
+
+Note that if not value is specified for the label or annotation, the filter will only check that exists.
+
+``` yaml
+kubernetes:
+  jobs:
+  - job_name_prefix: example
+    target_discovery: 
+      pod: true
```

Comment:
Maybe I'd comment the capability of using `pod: true` and `endpoints: true` at the same time and mentioning that 2 different scrape jobs will be added.

Reply:
yes I probably focus too much in the filter but didn't took into account the other sections of the target_discovery, i will add it thanks!